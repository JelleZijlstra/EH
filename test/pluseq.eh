#!/usr/bin/ehi
a = 0
printvar a
a += 1
printvar a

# error
b += 1
