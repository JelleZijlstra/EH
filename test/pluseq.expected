%%stdout%%
@int 0
@int 1
%%stderr%%
Uncaught exception of type NameError: <NameError: Unknown member b in object of type GlobalObject: (global execution context)>
